tests/show_test.py

```diff
@@ -49,3 +52,50 @@ def teardown_class(cls):
         print("TEARDOWN")
         os.environ["PATH"] = os.pathsep.join(os.environ["PATH"].split(os.pathsep)[:-1])
         os.environ["UTILITIES_UNIT_TESTING"] = "0"
+
+@patch('show.main.run_command')
+@pytest.mark.parametrize(
+        "cli_arguments,expected",
+        [
+            ([], 'cat /var/log/syslog'),
+            (['xcvrd'], "cat /var/log/syslog | grep 'xcvrd'"),
+            (['-l', '10'], 'cat /var/log/syslog | tail -10'),
+            (['-f'], 'tail -F /var/log/syslog'),
+        ]
+)
+def test_show_logging_default(run_command, cli_arguments, expected):
+    runner = CliRunner()
+    result = runner.invoke(show.cli.commands["logging"], cli_arguments)
+    run_command.assert_called_with(EXPECTED_BASE_COMMAND + expected, display_cmd=False)
+
+@patch('show.main.run_command')
+@patch('os.path.isfile', MagicMock(return_value=True))
+@pytest.mark.parametrize(
+        "cli_arguments,expected",
+        [
+            ([], 'cat /var/log/syslog.1 /var/log/syslog'),
+            (['xcvrd'], "cat /var/log/syslog.1 /var/log/syslog | grep 'xcvrd'"),
+            (['-l', '10'], 'cat /var/log/syslog.1 /var/log/syslog | tail -10'),
+            (['-f'], 'tail -F /var/log/syslog'),
+        ]
+)
+def test_show_logging_syslog_1(run_command, cli_arguments, expected):
+    runner = CliRunner()
+    result = runner.invoke(show.cli.commands["logging"], cli_arguments)
+    run_command.assert_called_with(EXPECTED_BASE_COMMAND + expected, display_cmd=False)
+
+@patch('show.main.run_command')
+@patch('os.path.exists', MagicMock(return_value=True))
+@pytest.mark.parametrize(
```

Comment:
Can you also add a test case to handle syslog.1 exists under the log.tmpfs directory as well?

Reply:
@gechiang, I have added the test case now.